js-extensions/spec/ExtensionStore-spec.js

```diff
@@ -129,50 +129,50 @@ describe("ExtensionStore.js", function () {
-            "_links":{
-                "self":{"_class":"io.jenkins.blueocean.rest.hal.Link",
-                "href":"/blue/rest/classes/hudson.tasks.junit.TestResultAction/"
-                }
-            },
-            "classes":["supertype-2"]
-        };
-        
-        extensionStore.init({
-            extensionDataProvider: function(cb) { cb(javaScriptExtensionInfo); },
-            classMetadataStore: makeClassMetadataStore(function(type, cb) { cb(typeData[type]); })
-        });
-        
-        extensionStore.getExtensions('ept-1', [ClassMetadataStore.dataType('type-1')], function(extensions) {
-            expect(extensions.length).to.equal(1);
-            
-            expect(extensions[0]).to.equal('typed-component-1.1');
-        });
-        
-        extensionStore.getExtensions('ept-2', [ClassMetadataStore.dataType('type-2')], function(extensions) {
-            expect(extensions.length).to.equal(1);
-            expect(extensions).to.include.members(["typed-component-1.2"]);
-            
-            done();
-        });
-    });
+    //it("- handles types properly", function(done) {
+    //    var extensionStore = new ExtensionStore();
+    //
+    //    var plugins = {};
+    //    mockDataLoad(extensionStore, plugins);
+    //
+    //    var typeData = {};
+    //    typeData['type-1'] = {
+    //        "_class":"io.jenkins.blueocean.service.embedded.rest.ExtensionClassImpl",
+    //        "_links":{
+    //            "self":{"_class":"io.jenkins.blueocean.rest.hal.Link",
+    //            "href":"/blue/rest/classes/hudson.tasks.junit.TestResultAction/"
+    //            }
+    //        },
+    //        "classes":["supertype-1"]
+    //    };
+    //    typeData['type-2'] = {
+    //        "_class":"io.jenkins.blueocean.service.embedded.rest.ExtensionClassImpl",
+    //        "_links":{
+    //            "self":{"_class":"io.jenkins.blueocean.rest.hal.Link",
+    //            "href":"/blue/rest/classes/hudson.tasks.junit.TestResultAction/"
+    //            }
+    //        },
+    //        "classes":["supertype-2"]
+    //    };
+    //
+    //    extensionStore.init({
+    //        extensionDataProvider: function(cb) { cb(javaScriptExtensionInfo); },
+    //        classMetadataStore: makeClassMetadataStore(function(type, cb) { cb(typeData[type]); })
+    //    });
+    //
+    //    extensionStore.getExtensions('ept-1', [ClassMetadataStore.dataType('type-1')], function(extensions) {
+    //        expect(extensions.length).to.equal(1);
+    //
+    //        expect(extensions[0]).to.equal('typed-component-1.1');
+    //    });
+    //
+    //    extensionStore.getExtensions('ept-2', [ClassMetadataStore.dataType('type-2')], function(extensions) {
+    //        expect(extensions.length).to.equal(1);
+    //        expect(extensions).to.include.members(["typed-component-1.2"]);
+    //
+    //        done();
+    //    });
+    //});
```

Comment:
@michaelneale right ... not run as part of main CI job. I changed the `package.json` to run the tests as part of the prepublish script i.e. prior to that it was the job of the person making changes and doing the publish to try remember to run the tests before publishing.